src/pages/geographies/[id].tsx

```diff
@@ -393,19 +397,24 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
         targets: [],
       };
     } else {
+      const { MCF, ...familyCountsWithoutMCF } = summaryInformation.family_counts;
+      const { MCF: mcfTopFamilies, ...topFamiliesWithoutMCF } = summaryInformation.top_families;
       return {
-        family_counts: summaryInformation.family_counts,
-        top_families: summaryInformation.top_families,
+        family_counts: familyCountsWithoutMCF,
+        top_families: topFamiliesWithoutMCF,
         targets: summaryInformation.targets,
```

Comment:
This was throwing off the numbers in the CCLW and CPR app because MCF was still beign passed
see example
![Screenshot 2024-10-31 at 12 01 07](https://github.com/user-attachments/assets/b743ab88-2ec9-44f7-be88-aad0a3e8f178)

Reply:
that extra 1 is the one MCF doc